railties/lib/rails/generators/rails/credentials/credentials_generator.rb

```diff
@@ -42,9 +42,14 @@ def credentials
         end
 
         def credentials_template
-          "# aws:\n#  access_key_id: 123\n#  secret_access_key: 345\n\n" +
-          "# Used as the base secret for all MessageVerifiers in Rails, including the one protecting cookies.\n" +
-          "secret_key_base: #{SecureRandom.hex(64)}"
+          <<-YAML.strip_heredoc
```

Comment:
Please add `require "active_support/core_ext/string/strip"`.

Reply:
`encrypted` command also has the same issue. Can you fix?
https://github.com/rails/rails/blob/52c9af6c51fbf686762ad2330f36fe9f8ce3c5d7/railties/lib/rails/generators/rails/encrypted_file/encrypted_file_generator.rb#L18

Reply:
@y-yagi Thanks for the comments. I've fixed them based on your advices.